src/runtime/pkg/resourcecontrol/cgroups.go

```diff
@@ -275,7 +277,7 @@ func (c *LinuxCgroup) Delete() error {
 	case cgroups.Cgroup:
 		return cg.Delete()
 	case *cgroupsv2.Manager:
-		if IsSystemdCgroup(c.ID()) {
+		if IsSystemdCgroup(c.ID()) && c.sandboxCgroupOnly {
```

Comment:
Whats about when we're using `cgroups.Cgroup` as the manager, does that also need the same fix?

Reply:
Maybe we need to do more tests, but i can't construct a case where `cgroups.Cgroup` acts as a manager in my environment.

Reply:
What is your environment? Try Ubuntu 20.04.

Reply:
After setting up an environment with Ubuntu 20.04, I will try it.